Formula/linkerd.rb

```diff
@@ -0,0 +1,82 @@
+class Linkerd < Formula
+  desc "Drop-in RPC proxy designed for microservices"
+  homepage "http://linkerd.io/"
+  url "https://github.com/BuoyantIO/linkerd/releases/download/0.7.5/linkerd-0.7.5.tgz"
+  sha256 "6f18f77b6dac019e24ccfb6adec74d6b13430be79af05f504461b39db85ebdca"
+
+  bottle :unneeded
+  depends_on java: "1.8+"
+
+  def install
+    # Replacing disco folder path
+    inreplace "config/linkerd.yaml", "disco", libexec/"disco"
+
+    # Installing and symlinking
+    libexec.install "disco", "linkerd-#{version}-exec"
+    share.install "docs"
+    pkgshare.install "config/linkerd.yaml"
+    bin.install_symlink libexec/"linkerd-#{version}-exec" => "linkerd"
+
+    # Configuring etc folder
+    mv "config", "linkerd"
+    etc.install "linkerd"
```

Comment:
Alternatively: `etc.install "config" => "linkerd"` which means the `mv` is unnecesary.